plugins/cue_points/quiz/lib/services/QuizUserEntryService.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
services directory should be directly under the plugin directory, not under lib, see other plugins.